packages/appstore/src/modules/tour-guide/tour-guide.tsx

```diff
@@ -126,17 +144,8 @@ const TourGuide = () => {
             floaterProps={{
                 disableAnimation: true,
             }}
-            callback={data => {
-                setJoyrideIndex(data.index);
-                if (data.action === 'prev' && data.index === tour_step_config_high_risk.length) {
-                    toggleIsTourOpen(false);
-                    setTimeout(() => {
-                        toggleIsTourOpen(true);
-                    }, 0);
-                }
-            }}
+            stepIndex={joyride_index}
```

Comment:
I believe having a callback prop for this component is essential rather than just hard coding some of the actions we wanted inside of the onclick function. You can refer to the documentation here: 
https://docs.react-joyride.com/callback